substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/SecurityServicesFeature.java

```diff
@@ -309,6 +310,17 @@ public void beforeAnalysis(BeforeAnalysisAccess access) {
             /* Resolve calls to com_sun_security_auth_module_UnixSystem* as builtIn. */
             PlatformNativeLibrarySupport.singleton().addBuiltinPkgNativePrefix("com_sun_security_auth_module_UnixSystem");
         }
+
+        if (isWindows()) {
+            access.registerClassInitializerReachabilityHandler(SecurityServicesFeature::registerSunMSCAPIConfig, clazz(access, "sun.security.mscapi.SunMSCAPI"));
```

Comment:
This should be a normal class reachability handler because the `sun.security.mscapi.SunMSCAPI` class is initialized at build time.
```suggestion
            access.registerReachabilityHandler(SecurityServicesFeature::registerSunMSCAPIConfig, clazz(access, "sun.security.mscapi.SunMSCAPI"));
```